libs/shared/ui/src/lib/components/inputs/input-text-small/input-text-small.tsx

```diff
@@ -27,15 +28,21 @@ export function InputTextSmall(props: InputTextSmallProps) {
     type = 'text',
     className = '',
     errorMessagePosition = 'bottom',
+    hasShowPasswordButton = false,
   } = props
 
   const [focused, setFocused] = useState(false)
+  const [_type, setType] = useState(type)
```

Comment:
I think it might be better to use, `currentType` and `setCurrentType` because we don't have this "style" of variable with "_"